src/test/resources/com/ibm/vCard/vcard_escaping.xml

```diff
@@ -6,7 +6,7 @@
     </Version>
     <Comment xmlns="">
       <Parameter xsi:type="xs:string">ENCODING=QUOTED-PRINTABLE</Parameter>
-      <Value xsi:type="xs:string">Testing escaping- : should appear &amp; ; comma should appear &amp; &#13;&#10; (CRLF) should appear &amp; / should appear, all in the same field </Value>
+      <Value xsi:type="xs:string">Testing escaping- : should appear &amp; ; comma should appear &amp; &#10; (LF) should appear &amp; / should appear, all in the same field </Value>
```

Comment:
Revert to CRLF

Reply:
So I believe that this is actually an issue due to XML and %CR; incompatibilities.  The expected data infoset should only expect an %LF; to be output instead of %CR;%LF;.

Reply:
Ah. Since this is a CRLF in the data, it ends up as an LF in the XML of the infoset. Barf.
https://daffodil.apache.org/infoset/ says that's what we do.
We are going to need a "preserve CR in data" option to map CR to %#xE00D; which is a private use area character, and that maps back to CR on output.  That or we have to make a custom XML creator that doesn't map CRLF to LF for us. The DFDL Infoset has no issue with carrying CR, LF, or CRLF, it's only when we project into XML that we encounter this nonsense.